src/Corporation/Corporation.ts

```diff
@@ -170,35 +183,55 @@ export class Corporation {
 
   determineCycleValuation(): number {
     let val,
-      profit = this.revenue - this.expenses;
+      assetDelta = (this.totalAssets - this.previousTotalAssets) / corpConstants.secondsPerMarketCycle;
+    // Handle pre-totalAssets saves
+    assetDelta ??= this.revenue - this.expenses;
```

Comment:
Since totalAssets and previousTotalAssets are members of the Corporation class which have default values of 150e9, I believe a loaded save that is missing these values will load in with 150e9 for each of these and the nullish conditional value isn't doing anything.

Even if totalAssets and previousTotalAssets were something like undefined, assetDelta would not be undefined but would instead be NaN, and the nullish assignment would do nothing.